source/_integrations/plugwise.markdown

```diff
@@ -236,33 +238,44 @@ script:
           preset_mode: asleep
 ```
 
-### Supported devices
+## Supported devices
+
+The Plugwise integration relies on the [plugwise](https://pypi.org/project/plugwise/) module for Python. It currently provides support for:
+
+### Adam
+
+A complete zone control system also known as Adam HA, supporting:
+
+- On/Off, OpenTherm or Loria/Thermastage heating and cooling support.
+- Running firmwares v3.x or v2.3
+- Additional devices:
+  - Zone thermostats such as Lisa or Anna (see warning below on Anna),
+  - A temperature sensor, Jip,
+  - Valve controllers called Floor or Tom,
+  - An under-floor heating controller Koen (note: a Koen always comes with a Plug, which is the active part),
+  - Smart switches, either Plug or Aqara Smart Plug.
+
+### Anna
 
-The current implementation of the Python module (Plugwise-Smile) includes:
+A smart thermostat, supporting:
 
-Adam (zone_control) with On/Off, OpenTherm, and Loria/Thermastage heating and cooling support:
+- OnOff, OpenTherm heating and Elga or Loria/Thermastage with heating and cooling support.
+- Running firmware v4.x, v3.x or v1.x.
 
- - v3.x
- - v2.3
+### P1 (DSMR)
 
- - Devices supported are Anna, Lisa, Jip, Floor, Tom, Plug, Aqara Smart Plug, and Koen (a Koen always comes with a Plug, the active part)
+A smart meter monitor for single or multiphase P1 and gas usage monitoring with the P1 running firmware v4.x, v3.x or v2.x.
 
-Anna (thermostat) with OnOff, OpenTherm heating, and Elga and Loria/Thermastage with heating and cooling support:
+### Stretch (end-of-sale)
 
- - v4.x
- - v3.x
- - v1.x
+For legacy power switches, such as the Circles or Stealths, with v3.x or v2.x Stretch firmware.
 
-On the Elga, the cooling-mode can only be turned on, or off, via a switch present on the device, not via a toggle in the Plugwise App.
-Please make sure to reload the Plugwise integration after the cooling-mode-switch is turned off after being on, or the other way around. This will ensure that the Plugwise integration is being adapted to the change in function of the Elga.
```

Comment:
Removing this for now, it will return as part of the 'known limitations' section already suggested and ready for upcoming PR (to keep this one small)